src/main/java/frc/robot/commands/DefaultMechCommand.java

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.LauncherConstants;
+import frc.robot.constants.AimingConstants.AimState;
+import frc.robot.constants.LauncherConstants.LauncherMode;
+import frc.robot.subsystems.AimingSubsystem;
+import frc.robot.subsystems.Input;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.LauncherSubsystem;
+import frc.states.MechState;
+import frc.states.MechStates.Intook;
+import frc.states.MechStates.ReadyForAim;
+import frc.states.MechStates.ReadyForHandoff;
+import frc.states.MechStates.ReadyForIntake;
+import frc.states.MechStates.ReadyForLaunch;
+
+public class DefaultMechCommand extends Command {
+    private IntakeSubsystem mIntakeSubsystem = new IntakeSubsystem();
```

Comment:
For all the fields, can you mark any field as `final` that can be?

Reply:
done, all member variables are either private final or private